src/lib/components/comparisonTable/components/Row/style.module.scss

```diff
@@ -116,7 +120,7 @@ h4.cell {
 h4.addon {
   border-right: none;
   max-width: calc(100vw - 64px);
-  width: 100vw;
+  width: 100&;
```

Comment:
Is this a typo?

Reply:
Yup, but if it works without I guess, then it's not even needed :D

Reply:
But it didn't work without it 😂